website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,288 @@
+
+## Navigating the interface
+
+The Visual Editor comprises a series of menus activated by clicking icons surrounding the border of the larger canvas. With none of the menu items activated, the workspace looks like this:
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-screen.png" width="90%" title="The Visual Editor canvas. The number of items is defined in this section." />
+
+Click on an icon to expand its section or execute an action depending on its purpose. The options are as follows:
+
+1. The model's title. This defaults to "Untitled" but can be edited anytime by clicking on it.
+2. The **Operators** toolbar contains the building blocks for creating a model with the editor. 
+3. The **SQL code** area displays the SQL that compiles your model.
+4. The **Runs and previews** that displays run data and previews data for individual operators.
+5. The **Commit history** display.
+6. The navigation tab that has icons for (from top to bottom):
+    - Zoom in
+    - Zoom out
+    - Center the model to fit the screen
+    - Zoom to selection (the operator selected on or off screen will be zoomed and centered)
+    - Auto-layout option for the individual operator tiles
+7. The [Copilot](/docs/cloud/dbt-copilot) icon (where available). Use natural language to build your dbt Visual Editor models.
+8. The **Run** command executes `dbt run` for the model.
+9. This button is initially a **Commit** command for your integrated Git provider. It will change to "Open pull request" once your first commit is made. The button will not initially appear until you begin working in the canvas area.
+
+## Create a model
+
+This section will walk you through creating a model with operators using sample data from the [Jaffle Shop](https://github.com/dbt-labs/jaffle-shop) project. With this guide, you will create a basic model that transforms two datasets to build a view of repeat customer purchases while you consider a loyalty program for your shop. 
+
+The operators are the heart of your model. They determine what data will be transformed and how. Click the **+** icon to open the operator menu.
+
+Operators are divided into three types:
+- **Input:** Input operators configure the source data.
+- **Transform:** Transform operators change and shape your data.
+- **Output:** Output operators define your model name and location.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/operators.png" width="90%" title="The operator’s menu on the side of the Visual Editor canvas." />
+
+Read more about the [individual operators](/docs/cloud/visual-editor-interface#operators) to understand the basic purpose of each. The dbt model created by the Visual Editor builds off of existing models. In this guide, there will be input (source) models and an output model (what you are building) which will be _your model_.  
+
+### Operator tiles
```

Comment:
This "Operator tiles" section feels like it could go after the procedure as a reference. I was beginning to wonder when we'd get to the steps. We could reference it in step 2 of "Create your model from pre-existing models" in a to "learn more about" sentence. This way for those who need more they can look at that section but others can forge ahead quickly.

Reply:
Good call out!